packages/backend/src/database/migrations/20210831103040_add_table_calculations_table.ts

```diff
@@ -0,0 +1,32 @@
+import { Knex } from 'knex';
+
+const TABLE_CALCULATIONS_TABLE_NAME =
+    'saved_queries_version_table_calculations';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(TABLE_CALCULATIONS_TABLE_NAME))) {
```

Comment:
Why do we need this check ?

Reply:
I default to `CREATE TABLE IF NOT EXISTS` just for safety (in case the migration runs twice etc. which has happened).

I originally had `knex.schema.createTableIfNotExists` but it threw a warning and told me to manually check for table existence before doing `knex.schema.createTable`